yaml/Intel/VT-d.yml

```diff
@@ -0,0 +1,1935 @@
+- name: VTD
+  short_description: VTD
+  children_name_with_prefix: VTD
+  type: group
+  fields:
+  - name: ROOT_ENTRY
+    description: |
+      The Root Table Address Register points to a
+      table of root-entries, when the Translation Table Mode (TTM) field in the register is 00b.
+    type: struct
+    size: 128
+    see: VTd[9.1(Root Entry)]
+    fields:
+    - size: 64
+      type: bitfield
+      name: 'Lower64'
+      fields:
+      - bit: 0
+        short_name: P
+        long_name: PRESENT
+        description: |
+          This field indicates whether the root-entry is present.
+
+          • 0: Indicates the root-entry is not present. All other fields are ignored by
```

Comment:
I would replace all the "•" dots with "*", to make it ASCII friendly, but maybe I'm too pedantic :)

Reply:
Did not you intend to make those "okay" when you added `_fix_text` ? 
 
The resulting header files will have `*` instead, and having to manually replace at the PDF->YML phase seem to contradict with the concept of this project. Although it does annoy me to0, a relaxed contribution rule helps gain more contributors.

Reply:
Ah, you're right, I totally forgot about that function. It's ok then.